templates/download/raspberry-pi/index.html

```diff
@@ -8,175 +8,146 @@
       </p>
-    </div>
-    <div class="col-5 u-hide--medium u-hide--small u-align--center">
-      {{
-        image(
-        url="https://assets.ubuntu.com/v1/016b770d-raspberry+pi+horizontal.svg",
-        alt="",
-        height="184",
-        width="148",
-        hi_def=True,
-        loading="auto"
-        ) | safe
-      }}
+      <div class="p-section">
+        {{
+          image(
+          url="https://assets.ubuntu.com/v1/016b770d-raspberry+pi+horizontal.svg",
+          alt="Raspberry Pi",
+          height="48",
+          width="36",
+          hi_def=True,
+          loading="auto"
+          ) | safe
+        }}
+      </div>
     </div>
   </div>
 </section>
 
-<section class="p-strip--light">
+<section class="p-section">
+  <hr class="p-rule is-fixed-width" />
   <div class="row">
     <div class="col-6">
       <h2>First time installing Ubuntu on Raspberry Pi?</h2>
+    </div>
+    <div class="col-6">
       <p>The simplest way is to use the <a href="https://www.raspberrypi.com/software/">Raspberry Pi Imager</a> which enables you to select an Ubuntu image when flashing your SD card.</p>
-      <p>If you are on Ubuntu, run:</p>
+      <p>If you are on Ubuntu, open the terminal and run:</p>
       <pre><code>sudo snap install rpi-imager</code></pre>
-      <p>And follow our <a href="/tutorials/how-to-install-ubuntu-desktop-on-raspberry-pi-4#1-overview">desktop</a>, <a href="/tutorials/how-to-install-ubuntu-on-your-raspberry-pi#1-overview">server</a> and <a href="/download/raspberry-pi-core">core</a> tutorials to get started.</p>
-      <p>For those not using the Raspberry Pi Imager, direct downloads are available below.</p>
-    </div>
-    <div class="col-6 u-align--center u-vertically-center">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/c4664fcd-raspberry-pi-imager.png",
-        alt="",
-        width="450",
-        height="303",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-      }}
-    </div>
-  </div>
-</section>
-
-<section class="p-strip is-shallow u-hide--small">
-  <div class="row">
-    <div class="col-8 col-medium-3 u-vertically-center">
-      <h2>Ready for Raspberry Pi 5</h2>
-    </div>
-    <div class="col-4 u-vertically-center u-align--center">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/953685e6-raspberry-pi-5.png",
-        alt="",
-        width="300",
-        height="200",
-        hi_def=True,
-        loading="auto"
-        ) | safe
-      }}
+      <p>And follow our <a href="/tutorials/how-to-install-ubuntu-desktop-on-raspberry-pi-4#1-overview">Desktop</a>, <a href="/tutorials/how-to-install-ubuntu-on-your-raspberry-pi#1-overview">Server</a> and <a href="/download/raspberry-pi-core">Core</a> tutorials to get started.</p>
+      <div class="p-image--wrapper p-section--shallow">
+        <img src="https://assets.ubuntu.com/v1/e24d92fc-raspberry-pi-imager-2024.png" />
```

Comment:
Please add this alt text: "Screenshot of the Raspberry Pi Imager, with three buttons letting you choose the Raspberry Pi device, operating system and storage where to write the image"